src/components/cylc/graph/Graph.vue

```diff
@@ -0,0 +1,631 @@
+              v-model="layoutName"
+              :items="layoutEngines"
+              label="Layout Engine"
+              @change="switchLayout"
+          ></v-select>
+          </v-expansion-panel-content>
+        </v-expansion-panel>
+      </v-expansion-panels>
+    </div>
+    <v-snackbar
+      top
+      right
+      absolute
+      color="red lighten-1"
+      :timeout=0
+      :value="warning"
+    >
+      WARNING: POC Graph View: beware of large workflows!
+      <v-btn
+        text
+        @click="warning = false"><v-icon>mdi mdi-close</v-icon></v-btn>
+    </v-snackbar>
+    <div class='cytoscape-navigator-overlay' ref="cytoscape-navigator-overlay">
+      <canvas></canvas>
+      <div class='cytoscape-navigatorView'></div>
+      <div class='cytoscape-navigatorOverlay'></div>
+    </div>
+  </div>
+</template>
+
+<script>
+import cytoscape from 'cytoscape'
+import dagre from 'cytoscape-dagre'
+import navigator from 'cytoscape-navigator'
+import panzoom from 'cytoscape-panzoom'
+import undoRedo from 'cytoscape-undo-redo'
+import popper from 'cytoscape-popper'
+import SyncLoader from 'vue-spinner/src/SyncLoader.vue'
+import Tippy from 'tippy.js'
+import nodeHtmlLabel from 'cytoscape-node-html-label'
+import { debounce, each, has } from 'lodash'
+import { mapState } from 'vuex'
+import {
+  states,
+  dagreOptions,
+  hierarchicalOptions,
+  panzoomDefaults,
+  cytoscapeDefaultStyle
+} from './index'
+
+export default {
+  name: 'Graph',
+
+  props: {
+    workflowName: {
+      type: String,
+      required: true
+    }
+  },
+
+  components: {
+    SyncLoader
+  },
+
+  data: function () {
+    return {
+      warning: true,
+      /**
+       * Loading indicator data. Used to tell the component whether it should display the loading status or not,
+       * which color to use, etc.
+       */
+      loading: true,
+      vueSpinner: {
+        color: '#5e9aff',
+        size: '.2em'
+      },
+      /**
+       * The Cytoscape layout options. By default it is set to Dagre. When switching the layout in the UI, it will
+       * replace this by a hierarchicalOptions or another options object.
+       */
+      layoutOptions: dagreOptions,
+      /**
+       * Available layout engines. These are used by components in the UI to allow the user to choose which layout
+       * to use in the UI
+       */
+      layoutEngines: [
+        'dagre',
+        'hierarchical'
+      ],
+      /**
+       * Cytoscape layout data. Controls things like the current layout, whether layout is frozen or not, etc.
+       */
+      layoutName: 'dagre',
+      layoutStopped: true,
+      layoutReady: false,
+      freeze: false,
+      /**
+       * Cytoscape tooltip plugin object.
+       */
+      tippy: null,
+      /**
+       * Graph data. These values are empty by default, then are populated later via watchers.
+       */
+      graphData: {
+        nodes: [],
+        edges: []
+      },
+      cytoscapeInstance: null
+    }
```

Comment:
Removed data that was not used, or javascript variables declared outside the data (i.e. not reactive, accessible from outside the component), and grouped them with some simple comments for future developers.